test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenTests.cs

```diff
@@ -43,6 +44,117 @@ public class JsonWebTokenTests
             new Claim("dateTimeIso8061", dateTime.ToUniversalTime().ToString("o", CultureInfo.InvariantCulture), ClaimValueTypes.DateTime, "LOCAL AUTHORITY", "LOCAL AUTHORITY"),
         };
 
+#if !NET452
```

Comment:
NET45? not NET452?

Reply:
@jennyf19 the test environment uses 452 as xunit is not available for 451.